wuzz.go

```diff
@@ -1009,25 +1018,27 @@ func (a *App) ParseArgs(g *gocui.Gui, args []string) error {
 			arg_index += 1
 			header := args[arg_index]
 			fmt.Fprintf(vheader, "%v\n", header)
-		case "-d", "--data":
+		case "-d", "--data", "--data-binary":
 			if arg_index == args_len-1 {
-				return errors.New("No POST/PUT value specified")
+				return errors.New("No POST/PUT/PATCH value specified")
 			}
 
-			vmethod, _ := g.View(REQUEST_METHOD_VIEW)
-			setViewTextAndCursor(vmethod, http.MethodPost)
-
 			arg_index += 1
-			add_content_type = true
+			set_data = true
+			set_binary_data = arg == "--data-binary"
 
-			data, _ := url.QueryUnescape(args[arg_index])
+			data := args[arg_index]
+			if !set_binary_data {
+				data, _ = url.QueryUnescape(data)
+			}
```

Comment:
Wouldn't be useful to add an `else` branch with `add_content_type = true` to set the content type if it isn't binary?

Also, I'm not sure if "binary" is the correct name for this feature, because this isn't really binary data, just URL unescaped. What about calling it "data-raw" instead of "data-binary"?